commit 895bef0568d3bf4ef6ce18d28d4204b6eddd24be
Author: bertin <macportsuser@macports.org>
Date:   Fri May 23 21:30:14 2025 +0200

    /opt/local/site-ports/sysutils/MacPorts-pextlib/files/lamarck/patch-no-enforced-universal-on-linux.diff

diff --git a/src/macports1.0/macports.tcl b/src/macports1.0/macports.tcl
index 091179ea..03f79a2a 100644
--- a/src/macports1.0/macports.tcl
+++ b/src/macports1.0/macports.tcl
@@ -3030,7 +3030,7 @@ proc macports::_upgrade_mport_deps {mport target} {
                                     ui_error "its dependency $dep_portname is only installed for the arch '$active_archs'"
                                     ui_error "and the configured universal_archs '$universal_archs' are not sufficient."
                                     return -code error "architecture mismatch"
-                                } else {
+                                } elseif {${macports::os_platform} eq "darwin"} {
                                     # upgrade the dep with +universal
                                     dict set variants universal +
                                     dict set options ports_upgrade_enforce-variants yes
